ui/l10n/en.js

```diff
@@ -425,6 +425,8 @@ var dictionary = {"ICMP.code":"ICMP Code",
 "label.allocated":"Allocated",
 "label.allocation.state":"Allocation State",
 "label.allow":"Allow",
+"label.annotated.by":"Annotator",
+"label.annotation":"Annotation",
```

Comment:
I do not see the gain in the rename. The word 'annotation' has connotations but so does 'comment'. Both are equally bad, IMHO.